### PR TITLE
Goodie Cheat Sheets section

### PR DESCRIPTION
**Pull request by talsraviv**
@zekiel @moollaza

### CODE REVIEW

File: duckduckhack/goodie/goodie_cheat_sheets.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# Goodie Cheat Sheets
+
+A popular (and perfect) use of Goodies is to create instant cheat sheets, available right from the DuckDuckGo search bar. To make adding a cheat sheet as quick as possible, we've brought all cheat sheets together under one Instant Answer, called the [Cheat Sheets Goodie](https://duck.co/ia/view/cheat_sheets).
+
+![vim cheat sheet](https://images.duckduckgo.com/iu/?u=https%3A%2F%2Fraw.githubusercontent.com%2Fduckduckgo%2Fduckduckgo-documentation%2Fmaster%2Fduckduckhack%2Fassets%2Fvim_cheat_sheet.png&f=1)
```
Comment (moollaza) [inline duckduckhack/goodie/goodie_cheat_sheets.md]: Hehe this is actually the old design, I'll send you a link to the new design...

File: duckduckhack/goodie/goodie_cheat_sheets.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+Instead of creating an entirely new Instant Answer, you simply need to add **one JSON file** to the existing [Cheat Sheets Goodie directory](https://github.com/duckduckgo/zeroclickinfo-goodies/tree/master/share/goodie/cheat_sheets).
+
+To do this in Codio, use the left-hand panel to enter the `zeroclickinfo-goodies` repository directory. Then click into the `share/goodie/cheat_sheets` directory. 
+
+In the file menu above, click **"Create New File"**, and enter the name of your cheat sheet as a JSON file (make sure it's saving in the `cheat_sheets` directory). For example, if the trigger for your cheat sheet is "gopro cheat sheet" create a new file called `gopro.json`.
+
+Erase any pre-filled contents, and enter the values for your cheat sheet using the [cheat sheet JSON syntax](#cheat-sheet-json-syntax).
+
+With this method, there is no need to create a new Instant Answer. There is also no need to edit the `CheatSheets.pm` file, `cheat_sheets.js`, or `cheat_sheets.css`. Simply save your new file, push the changes to your repository on github, and [submit a pull request](https://help.github.com/articles/creating-a-pull-request/).
```
Comment (moollaza) [inline duckduckhack/goodie/goodie_cheat_sheets.md]: nit: github -> GitHub

File: duckduckhack/goodie/goodie_cheat_sheets.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+### Cheat Sheet JSON Syntax
+
+Below is a summary of the [`vim.json`](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/share/goodie/cheat_sheets/vim.json) file, which displays a cheat sheet when searching for ["vim cheat sheet"](https://duckduckgo.com/?q=vim+cheat+sheet&ia=answer).
+
+All the top-level fields below are required. Syntax for the `key` property is explained below.
```
Comment (moollaza) [inline duckduckhack/goodie/goodie_cheat_sheets.md]: `metadata` is optional, only necessary if they have a useful place to send the user for further info/explanation, or if they're copying this stuff from a website

Comment (moollaza) [inline duckduckhack/goodie/goodie_cheat_sheets.md]: I guess `description` is technically optional as well, but encouraged

**Comment by zekiel** [comment]
nice - hoping the Lawrence meetup group can use this on Saturday. Please ping me (by air land and sea) if this doesn't get released by Friday afternoon.

**Comment by talsraviv** [comment]
All done on my end unless I missed something; @moollaza take a look

One question I have that I'm sure will come up - what if my cheat sheet name has two words? For example, "mortal kombat cheat sheet" - is that just `mortal_kombat.json`?

**Comment by moollaza** [comment]
LGTM :+1:

**Pull request merged**